src/NLU.DevOps.ModelPerformance.Tests/TestCaseSourceTests.cs

```diff
@@ -421,27 +440,26 @@ public static void ToEntityTestCasesWithFalseNegativeEntityValueMismatch()
         {
             var expectedUtterance = new LabeledUtterance(null, expected, null);
             var actualUtterance = new LabeledUtterance(null, actual, null);
+
             var compareResults = TestCaseSource.GetNLUCompareResults(
-                new[] { expectedUtterance },
-                new[] { actualUtterance });
+            new[] { expectedUtterance },
+            new[] { actualUtterance });
+
             compareResults.Statistics.Intent.TruePositive.Should().Be(truePositive);
             compareResults.Statistics.Intent.TrueNegative.Should().Be(trueNegative);
             compareResults.Statistics.Intent.FalsePositive.Should().Be(falsePositive);
             compareResults.Statistics.Intent.FalseNegative.Should().Be(falseNegative);
-            if (expected != null && expected != "None")
-            {
-                compareResults.Statistics.ByIntent[expected].TruePositive.Should().Be(truePositive);
-                compareResults.Statistics.ByIntent[expected].FalseNegative.Should().Be(falseNegative);
-            }
-            else if (actual != null && actual != "None")
-            {
-                compareResults.Statistics.ByIntent[actual].FalsePositive.Should().Be(falsePositive);
-            }
+
+            actual = actual ?? "null";
+            expected = expected ?? "null";
+            compareResults.Statistics.ByIntent[expected].TruePositive.Should().Be(truePositive);
+            compareResults.Statistics.ByIntent[expected].FalseNegative.Should().Be(falseNegative);
+            compareResults.Statistics.ByIntent[actual].FalsePositive.Should().Be(falsePositive);
         }
 
         [Test]
         [TestCase("foo", "foo", 1, 0, 0, 0)]
-        [TestCase(null, null, 0, 1, 0, 0)]
+        [TestCase(null, null, 0, 0, 0, 0)]
```

Comment:
Seems like this should still be a TruePositive?

Reply:
(If we consider the "None" intent as just any other intent, perhaps we should consider the "null" intent as any other intent)